text/0043-phrase-query.md

```diff
@@ -0,0 +1,172 @@
+- Title: Phrase Query
+- Start Date: 2021-06-02
+- Specification PR: [#43](https://github.com/meilisearch/specifications/pull/43)
+- MeiliSearch Tracking-Issues:
+
+# Phrase Query
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+MeiliSearch does not allow users a way to write a strict query in order to ask the engine to be more drastic in its selection of candidates for search results. The Phrase Query feature adds a simple syntax available to users to require the engine to select documents that will strictly match the search field value. That is, without typography, n-gram, wordsplit, prefix, synonym, and, proximity. In addition, the expression of a Phrase Query is case insensitive.
+
+### II. Motivation
+
+This feature is driven by user needs. Indeed, let's take an example recently brought up in our community slack.
+
+The user in question would like to be able to retrieve specifically the document containing the unique ISBN identifier and only that one. In a UX context of type as you search, this is impossible today without impacting the UI/UX or finding a workaround.
+
+- It could have had a specific search field only dedicated to this ISBN field in order to make it a filter and using the `=` operator. Not great for the UX.
+
+- It could also have kept a single search field and detected that an ISBN was entered in the search field using a regex to inject that filter at that point. Not great for the developper experience, moreover, what happens when a pattern cannot be specifically determined?
+
+The Phrase Query feature will easily solve this case but will also adapt to the needs of the user performing the search.
+
+### III. Additional Materials
+
+#### Algolia
+
+Algolia allows the use of Phrase Query syntax as long as the `advancedSyntax` parameter is set to true in the settings.
+
+As Algolia documentation said, a phrase query represents a specific sequence of terms that must be matched next to one another. A phrase query needs to be surrounded by double quotes ("). For example, the query "search engine" only returns a record if it contains “search engine” exactly in at least one attribute.
```

Comment:
```suggestion
As Algolia documentation said, a phrase query represents a specific sequence of terms that must be matched next to one another and in the given order. A phrase query needs to be surrounded by double quotes ("). For example, the query "search engine" only returns a record if it contains “search engine” exactly in at least one attribute.
```

Reply:
Not 100% sure that this is true for Algolia, but it would make more sense for an "exact" search than just requiring the query terms to be next to one another.